<launch>
  <arg name="serial_no_camera1"    default=""/> <!-- Note: Replace with actual serial number -->
  <arg name="serial_no_camera2"    default=""/> <!-- Note: Replace with actual serial number -->

  <group ns="camera1">
    <include file="$(find realsense2_camera)/launch/includes/nodelet.launch.xml">
      <arg name="serial_no"    value="$(arg serial_no_camera1)"/>
    </include>
  </group>

  <group ns="camera2">
    <include file="$(find realsense2_camera)/launch/includes/nodelet.launch.xml">
      <arg name="serial_no"    value="$(arg serial_no_camera2)"/>
    </include>
  </group>
</launch>
